docs/en/advanced/Scalability-Tuning.md

```diff
@@ -18,15 +18,17 @@ This page details the parameters to tune when scaling a cluster.
 ### Heap Size
 
 The Alluxio master heap size controls the total number of files that can fit into the master memory.
+If using the ROCKS off-heap metastore, the master heap size must be large enough to fit the inode cache.
+Provision roughly 1 KB of space for each inode.
 The following JVM options, set in `alluxio-env.sh`, determine the respective maximum heap sizes for the
 Alluxio master and standby master processes to `256 GB`:
 
 ```properties
-ALLUXIO_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
-ALLUXIO_SECONDARY_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
+ALLUXIO_MASTER_JAVA_OPTS+=" -Xms256g -Xmx256g "
+ALLUXIO_SECONDARY_MASTER_JAVA_OPTS+=" -Xms256g -Xmx256g "
 ```
 
-* As a rule of thumb set the min heap size to half the max heap size.
+* As a rule of thumb set the min and max heap size equal to avoid heap resizing.
```

Comment:
Is this always desired? I'm assuming this conclusion was drawn during some discussion I wasn't a part of.

Reply:
This is recommended when you "know what you are doing". (https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/sizing.html)
Since Alluxio is generally deployed on server-type machines I think the workloads are fairly predictable and the user can afford to provision a chunk of memory for the processes.

Reply:
Interesting, thanks!